### ISSUE TITLE
Notification - WCAG rule 2.2.1

### ISSUE THREAD

**Issue by SeamusLeonardHPE** [grommet/grommet#7610]
**Conditional rule**

2.2.1
Timing Adjustable
Level A

For each time limit that is set by the content, at least one of the following is true: 

- Turn off: The user is allowed to turn off the time limit before encountering it; or
- Adjust: The user is allowed to adjust the time limit before encountering it over a wide range that is at least ten times the length of the default setting; or
- Extend: The user is warned before time expires and given at least 20 seconds to extend the time limit with a simple action (for example, "press the space bar"), and the user is allowed to extend the time limit at least ten times; or
- Real-time Exception: The time limit is a required part of a real-time event (for example, an auction), and no alternative to the time limit is possible; or
- Essential Exception: The time limit is essential and extending it would invalidate the activity; or
- 20 Hour Exception: The time limit is longer than 20 hours.

---

**Notes**

Add an example for implementors on the DS site
